File: app/src/main/java/me/pwcong/jpstart/rxbus/event/GameEvent.java
package me.pwcong.jpstart.rxbus.event;

/**
 * Created by Pwcong on 2016/10/24.
 */

public class GameEvent {

    public static final int TYPE_PUZZLE = 1;
    public static final int TYPE_SUPPERZZLE = 2;

    private int type;

    public GameEvent(int type) {
        this.type = type;
    }

    @Override
    public String toString() {
        return "GameEvent{" +
                "type=" + type +
                '}';
    }

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }
}

File: app/src/main/java/com/github/pwcong/jpstart/rxbus/event/GameEvent.kt
package com.github.pwcong.jpstart.rxbus.event

class GameEvent(var type: Int) {
    override fun toString(): String {
        return "GameEvent{" +
                "type=" + type +
                '}'
    }

    companion object {
        const val TYPE_PUZZLE: Int = 1
    }
}
